charts/gcp-managed-cert/values.yaml

```diff
@@ -0,0 +1 @@
+domainName: ""
```

Comment:
I would much rather have a `values.schema.json` for every chart in `monster-helm`, and only have `values.yaml` for meaningful defaults.

Reply:
Defining it all in the values.yaml seems to be the community standard way of showing a user what can be overwritten (even if blank) in all the stable charts. This way someone can use `helm show values monster/<chart>` to see what is defined or able to be overwritten.

Reply:
I am ok with not replicating that standard. For the charts here, I'd much rather have the up-front validation that blocks even rendering a chart when values are missing.

Reply:
So are you asking me to delete values.yaml and just write a schema?

Reply:
Yes please 😄 Blame the Scala boi in me for wanting the static checks